DemoApp/ios/DemoApp/AppDelegate.m

```diff
@@ -20,10 +20,12 @@
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
 @import AppCenter;
+@import AppCenterAnalytics;
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AppCenterReactNative register];  // Initialize AppCenter
```

Comment:
do we need this line?

Reply:
The guide says that we need to open ios/YourAppName/AppDelegate.m and verify that we have [AppCenterReactNativeCrashes register]; instead of [AppCenterReactNativeCrashes registerWithAutomaticProcessing];

So looks like that addition of this line by the script is expected.

Reply:
Removed. This line has been added by script.